safe/report/test/test_impact_report.py

```diff
@@ -1157,7 +1156,7 @@ def test_minimum_needs_outputs(self):
                     'needs': [
                         {
                             'header': 'Toilets',
-                            'value': '0'
+                            'value': '10'
```

Comment:
do you know why it gives different values in 3.4 and 2.x?

Reply:
Different rounding mechanism from inside QGIS (or maybe Qt?).
To illustrate,
Our displaced count field is an integer. for some dataset that we test, in a certain aggregation boundary, the total count of affected is close to one, like 1.01.
With displacement ratio of 0.9, displacement count in double is 0.909.
But our field have a type of Qt Int.
In QGIS 2.x it gets rounded to 0.
Minimum needs are based on displaced count, meaning toilet min needs is 0.

In QGIS 3.x it gets rounded to the nearest integer, which is 1.
Minimum needs are based on displaced count, meaning toilet min needs is now 1.
In the report, as convention we round values below 10 to 10.

Reply:
But this is only apparent because the number is small though... For large counts, I don't think it will matter.

Reply:
Ah, I see. Perhaps the reason is the python 2 to 3 itself. See https://docs.python.org/3/library/functions.html#round

Anyway, good eye

Reply:
Yeah, it could be. I was horrified when seeing that this unittest fails because the value changed!
It should not happen because the process were deterministic, and people already uses InaSAFE 5.0 in QGIS. It has to be fixed ASAP.